pyproject.toml

```diff
@@ -39,7 +39,6 @@ fake-useragent = "^2.0.3"
 fastapi = "0.115.11"
 fastapi-versionizer = "4.0.1"
 faster-whisper = "1.1.1"
-flashrank = "^0.2.10"
```

Comment:
Can you elaborate on why removing this is A Good Thing?

Reply:
I'm still debating this myself. The idea is that better models already have good reranking capabilities, however, the impact of having this is so low that I don't think its worth removing it.